### DISCUSSION TITLE
HomeAssistant HomeKit Device Integration fails during setup

### DISCUSSION THREAD

**Discussion by 011V32** [rednblkx/HomeKey-ESP32#44] [Q&A] [answered]
I have set up a new system. After installation I added it to the WLAN.
Then it was found in HomeAssistant by HomeKit Device Integration.
I wanted to add it with this integration, however I got the following error and now it no longer shows up:
![image](https://github.com/rednblkx/HomeKey-ESP32/assets/22346481/c1126e7e-601a-4a78-86bf-b15dff52b0d5)
`This accessory is already paired to another device. Please reset the accessory and try again.`
That should work, right?

**Comment by rednblkx** [comment]
No, you can only pair one HomeKit device with one Controller. HASS is not actively searching for devices, it found the device once and it added it to the found devices list which I’m not sure when it’s cleared, I think that happens on a restart.
If you want to control it from hass you can checkout the dev branch, I added device integration with the hass through mqtt discovery a while back.

**Reply by 011V32** [reply]
But I haven't added it to any other controllers yet...
The [HomeKit Device](https://www.home-assistant.io/integrations/homekit_controller) Integration searches for HomeKit devices which are ready for pairing.
I do the same with other HomeKit devices. I add them to HomeAssistant using the [HomeKit Device](https://www.home-assistant.io/integrations/homekit_controller) Integration and then deploy them back to HomeKit using the [HomeKit Bridge](https://www.home-assistant.io/integrations/homekit) Integration. So I also have the HomeKit devices in HomeAssistant and HomeKit.

**Reply by rednblkx** [reply, marked as answer]
Either way it is out of scope for this project since homekey would only work when paired from Apple Home and the device and bridge integration does not even have support for nfc locks even though the underlying pyhap module has.

As I said before, if you want the device in hass you can try out the dev branch.
